django/core/management/commands/inspectdb.py

```diff
@@ -322,6 +319,12 @@ def normalize_col_name(self, col_name, used_column_names, is_relation):
 
         return new_name, field_params, field_notes
 
+    def table_to_model_name(self, table_name):
```

Comment:
Renamed to `normalize_table_name()` to make it consistent with `normalize_col_name()`.

Reply:
Thanks

Reply:
Good call was thinking the same thing 👍